### PR TITLE
tidy up gradle files, add configuration annotation

### PR DESCRIPTION
**Pull request by robzienert**

### CODE REVIEW

File: orca-keel/orca-keel.gradle [hunk 1]
```diff
@@ -1,25 +1,4 @@
...
-  compile spinnaker.dependency("spectatorApi")
-  compile spinnaker.dependency("frigga")
-  compile spinnaker.dependency('korkExceptions')
-
-  testCompile spinnaker.dependency("slf4jSimple")
-  testCompile 'org.spockframework:spock-unitils:1.1-groovy-2.4-rc-2'
-  testCompile 'org.codehaus.groovy:groovy-json:2.4.11'
+  compile project(":orca-retrofit")
+  testCompile project(":orca-test")
 }
```
Comment (robzienert) [inline orca-keel/orca-keel.gradle:4]: Most of this wasn't necessary.

File: orca-keel/src/main/groovy/com/netflix/spinnaker/orca/config/KeelConfiguration.java [hunk 2]
```diff
@@ -26,12 +26,14 @@
...
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
 import retrofit.Endpoint;
 import retrofit.Endpoints;
 import retrofit.RestAdapter;
 import retrofit.client.Client;
 import retrofit.converter.JacksonConverter;
 
+@Configuration
```
Comment (robzienert) [inline orca-keel/src/main/groovy/com/netflix/spinnaker/orca/config/KeelConfiguration.java:36]: This is what was preventing Travis from compiling.

Comment (emjburns) [reply orca-keel/src/main/groovy/com/netflix/spinnaker/orca/config/KeelConfiguration.java:36]: Arg thank you

File: orca-keel/orca-keel.gradle (hunk 1 above)
Comment (emjburns) [reply orca-keel/orca-keel.gradle:4]: Thank you!

Comment (robzienert) [reply orca-keel/orca-keel.gradle:4]: We'll revisit the kotlin stuff later, maybe on Monday if you're feeling well.

**Pull request merged**